app/experimenter/static/rapid/components/forms/ExperimentFormOptions.tsx

```diff
@@ -23,3 +23,30 @@ export const featureOptions = [
     description: "feature 2's description",
   },
 ];
+
+export const firefoxVersionOptions = [
+  { label: "Firefox 77.0", value: "77.0", description: "Firefox 77.0" },
```

Comment:
> Q1: How will this work when, say, Firefox 83 is current Release channel? Would we still have older ex-Release versions listed at the top of the `<select>` menu?

The list is static right now so the options presented won't change unless we explicitly change them.  Is that what you mean?

> Q2: Do we have to modify this menu after every new release is released?

Well right now it goes up to 100, so I guess when we get close to that we'll dump a bunch more options in.  Why would we need to do it after every release?

> Q3: Does the `<select>` menu preselect the current Release channel version?

To do that we'd have to have knowledge of the release calendar, which would require a bunch of work to pull it in from somewhere and update the UI with it, etc.  @AnaMedinac asked for the same thing, but we haven't done anything like that yet.

Reply:
Currently looks like:

![Mozilla_Experimenter](https://user-images.githubusercontent.com/557895/87593202-f0c89b80-c69f-11ea-8312-8d2d4bbe62ab.png)

I think my concerns are if Firefox 83 is release, we'd have ~6 old builds at the top of the list that don't make sense. Or each new experiment would always have to select the correct Release version from a dropdown. Or if you mistakenly select "Firefox 94.0" from the dropdown (because reasons), the experiment would launch (if not caught+rejected by an operator), but nothing would happen because even after your enrollment+test period, that version would never be in scope and you'd presumably get no results back.

But yeah, maybe its too tricky to scrape the list of current trains from https://whattrainisitnow.com/ (or the upstream https://product-details.mozilla.org/1.0/firefox_versions.json which is where they apparently fetch from). Having a pseudo-dynamic list of dropdown values adds a lot of weird edges.
I just didn't know if a select list with only 3 options ("**Release** _Firefox 78.0_", "**Beta** _Firefox 79.0_", and "**Nightly** _Firefox 80.0_") would make it easier an easier UX for experiment submitters. Since we only ever show 3 options, "Release" would always be the first/default option and what submitters would want 99.99% of the time anyways.

Reply:
@pdehaan Yeah these are all very valid and good points, I'd suggest taking it out of here and moving exactly what you just wrote into https://github.com/mozilla/experimenter/issues/2948, which I just reopened, so that we can think through it in more detail.  Also thank you for the detailed write up!